### ISSUE TITLE
Corporations Canada filing by Jul 31

### ISSUE THREAD

**Issue by StephMcNeely** [ForetCapitaleForest/actions#90]
The 2022 annual return for Forêt Capitale Forest (Corporation Number: 1208866-5) must be filed between 2022-06-01 and 2022-07-31.

- [ ] get board review of Financial statements at June 29 meeting
- [ ] provide updates to Greenbooks by July 5

**Comment by smlambert** [comment]
@StephMcNeely - would this code have been mailed to my place?  I do not believe I have this code sent by email... will connect with you next week to figure it out.

**Comment by StephMcNeely** [comment]
Yes, I believe it would have come by mail.  Thanks - let me know what you find.  
I can apply for a new code but not urgent.  It's to update our FCF officers list
